### PR TITLE
Split tourism_activity layer

### PR DESCRIPTION
**Pull request by yjacolin**
* Split in two layers: tourisme_activity and informations (already there before
splitting)
* Add new activities
* I18n

### CODE REVIEW

**Comment by yjacolin** [comment]
@eleu, @marionb or @tsauerwein if you can check the "de" translation :)

File: mapserver/osm.map.mako [hunk 1]
```diff
@@ -200,7 +195,7 @@ LAYER
     PROCESSING "CLOSE_CONNECTION=DEFER" # For performance
     CONNECTION "user=${dbuser} password=${dbpassword} host=${dbhost} dbname=osm"
     DATA "geom FROM (SELECT regexp_replace(format(\'%s\', name), \'^$\', osm_id::text) AS display_name,name,osm_id,access,aerialway,amenity,barrier,bicycle,brand,building,covered,denomination,ele,foot,highway,layer,leisure,man_made,motorcar,\"natural\", operator, population, power, place, railway, ref, religion, shop, sport, surface, tourism, waterway, wood, way AS geom FROM planet_osm_point) AS foo USING UNIQUE osm_id USING srid=21781"
-    PROCESSING "NATIVE_FILTER=tourism in ('zoo','information','museum','viewpoint')"
+    PROCESSING "NATIVE_FILTER=tourism ='information'"
```
Comment (sbrunner) [inline mapserver/osm.map.mako]: missing space :-)

**Comment by sbrunner** [comment]
Looks good :-)

**Comment by yjacolin** [comment]
thanks @sbrunner waiting for a feedback from our german speaking workmet :)

**Comment by tsauerwein** [comment]
:+1:

File: demo/static/js/Proj/Lang/de.js [hunk 2]
```diff
@@ -42,6 +42,8 @@ OpenLayers.Util.extend(OpenLayers.Lang.de, {
     "entertainment": "Unterhaltung",
     "sustenance": "Nahrung",
     "Hobbies": "Hobbys",
+    "accommodation": "Unterkunft",
+    "tourism_activity": "Tourismus-Aktivität",
```
Comment (marionb) [inline demo/static/js/Proj/Lang/de.js]: `Touristenaktivität` is probably the better word.

File: demo/static/js/Proj/Lang/de.js [hunk 3]
```diff
@@ -42,6 +42,8 @@ OpenLayers.Util.extend(OpenLayers.Lang.de, {
...
```
Comment (marionb) [inline demo/static/js/Proj/Lang/de.js]: Touristenaktivität is probably the better word.

**Pull request merged**